src/main/java/com/uid2/shared/attest/AttestationTokenRetriever.java

```diff
@@ -96,16 +96,28 @@ public AttestationTokenRetriever(Vertx vertx,
         }
         this.appVersionHeader = builder.toString();
 
+        this.vertx.setPeriodic(0, attestCheckMilliseconds, this::attestationCheck);
```

Comment:
Why have you moved this to the constructor? Currently it is only set after the first attestation, but now it will happen 60s after the construction, regardless of whether attest has been called.

Reply:
changed back